aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -144,6 +144,24 @@ pub enum EntryFunctionCall {
         coin_type: TypeTag,
     },
 
+    /// `Sponsor` can add locked coins for `recipient` with given unlock timestamp (in seconds).
+    /// There's no restriction on unlock timestamp so sponsors could technically add coins for an unlocked time in the
+    /// past, which means the coins are immediately unlocked.
+    LockedCoinsAddLockedCoins {
```

Comment:
Do we want framework SDK generating function for move-example now?

Reply:
This seems to have always been here. If you think we should remove, we can discuss this separately :)